cli/sawtooth_cli/cluster.py

```diff
@@ -301,6 +300,10 @@ def do_cluster_start(args):
 
     save_state(state)
 
+    # Make sure the docker cluster network is up before starting nodes
+    if isinstance(node_controller, DockerNodeController):
```

Comment:
Use of isinstance() like this is an antipattern.  It may be better to always call the method on the controller but make the implementation on the non-docker ones "pass" (which could be done in the base class by default).